validation/src/main/scala/hmda/validation/rules/lar/validity/V300.scala

```diff
@@ -0,0 +1,66 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.census.CBSATractLookup
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.PredicateRegEx._
+
+object V300 extends EditCheck[LoanApplicationRegister] {
+
+  val cbsaTracts = CBSATractLookup.values
+
+  val validCombination = cbsaTracts.map { cbsa =>
+    (cbsa.geoIdMsa, cbsa.state, cbsa.county, cbsa.tractDecimal)
+  }
+
+  val validMdCombination = cbsaTracts.map { cbsa =>
+    (cbsa.metDivFp, cbsa.state, cbsa.county, cbsa.tractDecimal)
+  }
+
+  val validStateCountyCombination = cbsaTracts.map { cbsa =>
+    (cbsa.state, cbsa.county, cbsa.tractDecimal)
+  }
```

Comment:
I'm not familiar enough with the CBSA data file to know whether there will be any duplicates in `validStateCountyCombination`, `validMdCombination`, or `validCombination`. If so, adding `.distinct` to the end would improve performance at runtime.